common/utils/src/main/scala/org/apache/spark/internal/LogKey.scala

```diff
@@ -21,5 +21,9 @@ package org.apache.spark.internal
  * All structured logging keys should be defined here for standardization.
  */
 object LogKey extends Enumeration {
-  val EXECUTOR_ID, MIN_SIZE, MAX_SIZE = Value
+  val APPLICATION_ID, APPLICATION_STATE, BUCKET, CONTAINER_ID, EXECUTOR_ID, POD_ID = Value
+  val MIN_SIZE, MAX_SIZE, MAX_EXECUTOR_FAILURES = Value
+  val EXIT_CODE = Value
+
+  type LogKey = Value
```

Comment:
why do we need this?

Reply:
Because I want to use 'xxx: LogKey' instead of 'xxx: LogKey.Value' to define `a type`, it feels like it's a `personal preference`. Should we `restore` it?
https://github.com/apache/spark/blob/c3da26051d0c4c957e1bca98f31fcd575d3a1054/core/src/main/scala/org/apache/spark/scheduler/TaskLocality.scala#L23-L32